### PR TITLE
[TC-8998] Update the FTP connector pages

### PR DESCRIPTION
**Pull request by marcmatt**
https://tradecloud.atlassian.net/browse/TC-8998
https://tradecloud.atlassian.net/browse/TC-8950

Please review https://docs.tradecloud1.com/connectors/~/revisions/7vmJCJqrRJ6hmayBOLGr/ftp-connectors

### CODE REVIEW

File: ftp-connectors/README.md [hunk 1]
```diff
@@ -23,45 +20,61 @@ Your data format and transport can be set in your company integration settings b
...
 ## FTP Flows
 
 ### Uploading a file to Tradecloud
 
+{% hint style="warning" %}
+Both parties must use `0664` permission when uploading files.
+Setting the correct permissions is the responsibility of the client as outlined in the [standard](https://datatracker.ietf.org/doc/html/draft-ietf-secsh-filexfer-13#section-7.6).
+{% endhint %}
+
 ![](../.gitbook/assets/20180218-ftp-upload-flow-by-customer.png)
```
Comment (ghost) [inline ftp-connectors/README.md]: Technically speaking, it's the polling that happens _after_ the file was uploaded that will cause step 3 and 4

File: ftp-connectors/README.md [hunk 2]
```diff
@@ -23,45 +20,61 @@ Your data format and transport can be set in your company integration settings b
...
-| :--- | :--- | :--- | :--- |
-| Tradecloud | Periodic polling | order\_response | Periodically poll the folder for a new file |
-| Customer | 1. Export file |  | Customer  exports a new file |
-| Customer | 2. Upload file | order\_response | Customer uploads a new file |
-| Tradecloud | 3. Download file | order\_response | Tradecloud downloads a new file |
-| Tradecloud | 4. Import file |  | Tradecloud imports the file |
-| Tradecloud | 5. Archive file. | archive/order\_response | When successfully imported,  move the file to archive |
+| Party | Step | Description |
+| :--- | :--- | :--- |
+| Tradecloud | Periodic polling |Periodically poll the folder for a new file |
```
Comment (vovinacci) [inline ftp-connectors/README.md]: Nit

```suggestion
| Tradecloud | Periodic polling | Periodically poll the folder for a new file |
```

File: ftp-connectors/csv-ftp-connector.md [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
 | Data Format | [RFC 4180](https://tools.ietf.org/html/rfc4180) |
-| Message types | Order \(new and changed\), Order response \(new and changed\) |
-| Specification | Not available yet |
-| Transport | SFTP and FTPS server at Tradecloud side |
-| Requirements | SFTP or FTPS client at your side, outbound internet access. |
-| Availability | Not available yet \(use the [CSV file connector](https://docs.tradecloud1.com/connectors/cvs-file-connector) in the meanwhile\) |
+| Message types | Order, Order update, Order Response, Despatch Advice, Receipt Advice |
+| Specification | Custom made |
+| Transport | [SSH File Transfer Protocol](https://datatracker.ietf.org/doc/html/draft-ietf-secsh-filexfer-13) |
+| Host |SFTP server at Tradecloud side |
```
Comment (vovinacci) [inline ftp-connectors/csv-ftp-connector.md]: Nit

```suggestion
| Host | SFTP server at Tradecloud side |
```

File: ftp-connectors/edifact-ftp-connector.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
-| Specifications | Not available yet |
-| Transport | SFTP and FTPS server at Tradecloud side |
-| Requirements | SFTP or FTPS client at your side, outbound internet access. |
-| Availability | Not available yet |
-
+| Data Format | UN/EDIFACT D.96A |
+| Message types | ORDERS, ORDCHG, ORDRSP, DESADV, RECADV |
+| Specifications | [UN/EDIFACT D.96A zip download](https://unece.org/DAM/trade/untdid/d96a/d96a.zip) |
+| Transport | [SSH File Transfer Protocol](https://datatracker.ietf.org/doc/html/draft-ietf-secsh-filexfer-13) |
+| Host |SFTP server at Tradecloud side |
```
Comment (vovinacci) [inline ftp-connectors/edifact-ftp-connector.md]: Nit

```suggestion
| Host | SFTP server at Tradecloud side |
```

**Review by vovinacci — APPROVED** [review]
LGTM with few nits

**Review by ghost — APPROVED** [review]
Looks great!

File: ftp-connectors/README.md (hunk 1 above)
Comment (marcmatt) [reply ftp-connectors/README.md]: Improved

**Pull request merged**